ddcz/templates/users/sign_up.html

```diff
@@ -0,0 +1,131 @@
+{% extends "public.html" %}
+
+{% block styles %}
+<link rel="stylesheet" href="{{ reg_style }}">
+{% endblock %}
+
+{% block scripts %}
+<script type="text/javascript" src="{{ reg_script }}" async></script>
+{% endblock %}
+
+
+{% block content %}
+<h1 class="page-heading">Vstup do města Dračí Doupě</h1>
+        {% if sign_up_form.errors %}
+            {% for field in sign_up_form %}
+                {% for error in field.errors %}
+                    <div>
+                        <strong>{{ error|escape }}</strong>
```

Comment:
I *think* that everything is escaped by default unless explicitly marked as safe, unless we changed it somewhere. Have you tested whether it's needed?

Reply:
You're right, I removed the escape tag.